CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+We use the [GitHub Flow](https://guides.github.com/introduction/flow/) in this project. Specific guidelines are discussed below.
+ 
+## Creating Changes
+
+1. Start a new branch from `master` and name it according to the relevant user story. The format is `#-name`. Example:
+    
+    ```bash
+    git checkout master
+    git checkout -b 15-create-contributing-guidelines
+    ```
+2. Make some modifications and commit the changes.
+ 
+    ```bash
+    git add CONTRIBUTING.md
+    git commit -m "Update CONTRIBUTING.md"
+    ```
+
+3. Push the new branch.
+
+    ```bash
+    git push -u origin 15-create-contributing-guidelines
+    ```
+
+4. Create some more changes and push them.
+
+    ```bash
+    git push
+    ```
+
+5. When the feature is done create a new Pull Request (PR) using the GitHub web interface.
```

Comment:
Sounds good. Would adding a [template](https://github.com/blog/2111-issue-and-pull-request-templates) make sense?